modules/c2c/murano2cloud.lua

```diff
@@ -4,25 +4,48 @@ local murano2cloud = {}
 
 murano2cloud.alias = "openweatherapi"
 
-function murano2cloud.addIdentity(identity)
+local function getQuery(identity)
+  local lon, lat = string.match(identity, '^([0-9.]+),([0-9.]+)$')
```

Comment:
@sauvainr This regex is not happy with the real GPS example I put, but the following works from the regex tool
```
^([0-9.]*),([0-9.]*)
```

My Example:
```
121.1837289,25.0344194
```

Reply:
The match works.. maybe you had an extra space? I loosed a bit the regex to parse if have a some front or back space.